tests/datasets/test_spacenet.py

```diff
@@ -223,6 +223,117 @@ def test_plot(self, dataset: SpaceNet4) -> None:
         plt.close()
 
 
+class TestSpaceNet5:
+    @pytest.fixture(params=["PAN", "MS", "PS-MS", "PS-RGB"])
```

Comment:
Let's combine these two fixtures into a single fixture. Your `request.param` can be a list of attributes, so you can use `zip` or `itertools.product` or something like that to add more combinations.

Reply:
Done